packages/backend/src/routes/attesterDescription.mjs

```diff
@@ -0,0 +1,142 @@
+import { ethers } from 'ethers'
+import fetch from 'node-fetch'
+import catchError from '../helpers/catchError.mjs'
+import { APP_ADDRESS, ARBITRUM_API_KEY, wallet } from '../config.mjs'
+import { attesterDescriptionAbi } from '../helpers/abi.mjs'
+import axios from 'axios'
+
+export default ({ app, db, synchronizer }) => {
+  const handleSet = async (req, res) => {
+    // EIP Interface code
+    // isValidSignatureSelector 0xe0c5e6c3
+    // getDescriptionSelector 0x1a092541
+    // setDescriptionSelector 0xf0d3533b
+    // interfaceId 0x93c93c46
+
+    // attesterId -> contgract
+    // signature -> deployer
+
+    const attesterId = req.params.attesterId
+    const { icon, url, name, description, nonce, signature, network } =
+      req.headers
+    const _id = attesterId + network
+
+    let passed = true
+
+    const contract = new ethers.Contract(
+      APP_ADDRESS,
+      attesterDescriptionAbi,
+      wallet
+    )
+
+    const validUrl = await fetch(url).catch(() => false)
+
+    if (!validUrl) {
+      res.status(401)
+      passed = false
+    }
+
+    let supportsInterface = false
+    try {
+      supportsInterface = await contract.supportsInterface('0x93c93c46')
+    } catch (_) {
+      // assume the function call fails and the interface is not supported
+      res.status(401)
+      passed = false
+    }
+
+    const hash = ethers.utils.solidityKeccak256(
+      ['uint256', 'string'],
+      [nonce, description]
+    )
+
+    const _res = await axios({
+      url: 'https://api-goerli.arbiscan.io/api?',
```

Comment:
it could be from different network
so I think the router should handle different network parameters
e.g. it has been deployed to different networks: https://developer.unirep.io/docs/testnet-deployment

Reply:
added a util with a test for handling a bunch of networks